src/utils.ts

```diff
@@ -21,6 +21,14 @@ export const filterOutByGroupId = (pendingMessages: PendingMessages, msg: Messag
     )
 }
 
+export const getMessagesByGroupId = (pendingMessages: PendingMessages, msg: Message): PendingMessages => {
+    return pendingMessages.filter(
+        (m) =>
+            m.sqsMessage.Attributes?.MessageGroupId != null &&
```

Comment:
Can this be issue if messages have no groupId? 🤔 I don't know if it can't cause some issues with standard queues etc. probably not

Reply:
It shouldn't imho. If there is no group id the message will be ignored in both cases why we're doing this.